### PR TITLE
NAS-102996: Add nat_forwards field to docs

### PR DESCRIPTION
**Pull request by Mrt134**
- Add intro entry
- Add entry to Jails/Network Properties table with draft description.
- HTML build test: no issues.

### CODE REVIEW

**Comment by Mrt134** [comment]
I'm not really satisfied with the description I've got now, but am hitting a wall when trying to word it differently. I need to open a PR in webui to create help text for the nat_forwards field when the description is finalized here.

File: userguide/jails.rst [hunk 1]
```diff
@@ -624,6 +624,12 @@ Click :guilabel:`NEXT` to view all jail
    |                        |              | addresses, enter the host MAC address and the jail MAC address separated by a space.                    |
    |                        |              |                                                                                                         |
    +------------------------+--------------+---------------------------------------------------------------------------------------------------------+
+   | nat_forwards           | checkbox     | `Network Address Translation (NAT) port forwarding <https://en.wikipedia.org/wiki/Port_forwarding>`__   |
```
Comment (wblock) [inline userguide/jails.rst]: Shouldn't this be _NAT Forwards_, or are we still trying to stay with the same names as iocage fields?

File: userguide/jails.rst [hunk 2]
```diff
@@ -624,6 +624,12 @@ Click :guilabel:`NEXT` to view all jail
...
+   |                        |              | options. Setting for :guilabel:`NAT` jails adds :guilabel:`protocol`, :guilabel:`jail_port`, and        |
```
Comment (wblock) [inline userguide/jails.rst]: s/Setting for/Enabling on/

File: userguide/jails.rst (hunk 1 above)
Comment (Mrt134) [reply userguide/jails.rst]: This is what it is in the UI right now. I don't doc what it isn't

File: userguide/jails.rst [hunk 3]
```diff
@@ -624,6 +624,12 @@ Click :guilabel:`NEXT` to view all jail
...
+   |                        |              | :guilabel:`host_port` port forwarding options. Additional port forwarding configurations can be created |
+   |                        |              | by clicking |ui-add|. Multiple jails can set the same :guilabel:`jail_port` value, but only             |
```
Comment (wblock) [inline userguide/jails.rst]:
```suggestion
   |                        |              | by clicking |ui-add|. Multiple jails can be set to use the same :guilabel:`jail_port`, but only         |
```

File: userguide/jails.rst [hunk 4]
```diff
@@ -624,6 +624,12 @@ Click :guilabel:`NEXT` to view all jail
...
+   |                        |              | one jail can be running with this configuration.                                                        |
```
Comment (wblock) [inline userguide/jails.rst]:
```suggestion
   |                        |              | one running jail can use an individual port.                                                        |
```

**Pull request merged**